platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -225,7 +225,7 @@ public void run() {
    */
   public void mergeOfflineRegions(@NonNull String path, @NonNull final MergeOfflineRegionsCallback callback) {
     final File src = new File(path);
-    new FileUtils.CheckFileReadPermissionTask(new FileUtils.OnCheckFileReadPermissionListener() {
+    final FileUtils.OnCheckFileReadPermissionListener fileReadPermissionListener = new FileUtils.OnCheckFileReadPermissionListener() {
```

Comment:
We can revert this change back if needed (that was from previous testing / debugging)